### PR TITLE
Fixed player death event broadcaster

### LINKED ISSUE #219: NitroxPatcher fails to build
I've been trying to build Nitrox a few times now, and every time I get the same build-error.

`1>------ Build started: Project: NitroxPatcher, Configuration: Debug Any CPU ------
1>V:\tempo\Nitrox-master\Documents\Nitrox\NitroxPatcher\Patches\TelemetryReporting_SendPlayerDeathEvent_Patch.cs(12,59,12,77): error CS0246: The type or namespace name 'TelemetryReporting' could not be found (are you missing a using directive or an assembly reference?)
2>------ Build started: Project: NitroxTest, Configuration: Debug Any CPU ------
========== Build: 0 succeeded, 2 failed, 6 up-to-date, 0 skipped ==========`

And sure enough, the thing that holds it back is indeed the reference to TelemetryReporting, which it claims doesn't exist.

This has been done with the newest repo pulled.

Error specifically found in "TelemetryReporting_SendPlayerDeathEvent_Patch.cs" on line 12.

Any assistance would be deeply appreciated.

### PR DESCRIPTION
**Pull request by ahk1221**
Fixes #219

### CODE REVIEW

File: SharedConfig.targets [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <DevVars>DevVars.targets</DevVars>
     <DevVarsLoc>$(SolutionDir)\$(DevVars)</DevVarsLoc>
-    <SubnauticaDir Condition="!Exists($(DevVarsLoc))">C:\Program Files (x86)\Steam\steamapps\common\Subnautica</SubnauticaDir>
+    <SubnauticaDir Condition="!Exists($(DevVarsLoc))">D:\Steam Library\steamapps\common\Subnautica</SubnauticaDir>
```
Comment (MarijnS95) [inline SharedConfig.targets]: Please don't change the default location; for this you can rename `DevVars.targets.example` to `DevVars.targets` and specify the location there.

File: .gitignore [hunk 2]
```diff
@@ -274,3 +274,4 @@ paket-files/
 # JetBrains Rider
 .idea/
 *.sln.iml
+*.targets
```
Comment (MarijnS95) [inline .gitignore]: These files are part of the project and should not be ignored. The local `DevVars.targets` file is already explicitly ignored.

**Comment by ahk1221** [comment]
Fixes #219

**Comment by ZiotecH** [comment]
Thank you for fixing this, really appreciate it.
Keep up the good work!

**Pull request merged**